translations/en-us.yml

```diff
@@ -1665,11 +1665,10 @@ registries:
             done_add_new_button: 'Done adding new contributors'
             results_heading: 'Results'
             search_placeholder: 'Search by name or profile information'
-            help_text: '<p>Search results will appear here. Click the + icon in each row to set permissions for that contributor.</p><p>You can perform additional searches to add more contributors. Your selections will remain listed below until you click Save.</p>'
             search: 'Search'
-            add_unregistered_contributor: 'Add unregistered contributor'
+            add_unregistered_contributor: 'Add by email address'
```

Comment:
Not a problem with your code, but I worry this language may be a bit misleading. I would think this would mean we are looking for an OSF user by email (which feels like a reasonable identifier to find users by). I'll ask Mark if this is something to consider

Reply:
Isn't there the case where you can add the contributor without an email address, just a full name, and if someone claims it, the OSF will send an email to the admins asking if that's the correct person?